### DISCUSSION TITLE
An error occurs in the rpc module.

### DISCUSSION THREAD

**Discussion by songminkyu** [cefsharp/CefSharp#4888] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

126.2.180

### Operating System

Windows 10

### Architecture

x64

### .Net Version

net 8.0

### Implementation

WPF

### Reproduction Steps

hello

If you enable native debugging in C# and run it after building, the following error occurs.

Unhandled exception in FINALForensics.exe at 0x00007FFCB8B230C4 (rpcrt4.dll). 0xC0020043: rpc encountered an internal error (parameters: 0x0000000000000001, 0x00000000000006A6).

However, the problem does not occur in CefSharp.Wpf.NETCore version 124.3.80, so what is the difference?

The development environment is net 8.0 and WPF, and the Cefsharp version is 126.2.180.

Cef.Initialize(settings);

It says that an rpc exception occurs after the call.

### Expected behavior

**124.3.80 worked without problem.
126.2.180 A problem occurred after upgrading.**

  namespace FD.CefServices
  {
      public class CefService
      {
          public CefService() { }
  
          /// <summary>
          /// Please customize CefSettings.root_cache_path for your application. Use of the default value may lead to unintended process singleton behavior.
          /// </summary>
          public void Cef_InitializeRootCachePath()
          {            
              var settings = new CefSettings();            
           
              string createdatetime = DateTime.Now.ToString("yyyyMMdd_HHmmss");
              string RootCefCachePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Temporary Data", "CefCache", createdatetime);
              if (!Directory.Exists(RootCefCachePath))
              {
                  Directory.CreateDirectory(RootCefCachePath);
              }
  
              settings.RootCachePath = RootCefCachePath;
        
              Cef.Initialize(settings);
             
          }
          public void Cef_Shutdown()
          {
              Cef.Shutdown();
          }
      }
  }

### Actual behavior

Unhandled exception in FINALForensics.exe at 0x00007FFCB8B230C4 (rpcrt4.dll). 0xC0020043: rpc encountered an internal error (parameters: 0x0000000000000001, 0x00000000000006A6).

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

**Comment by amaitland** [comment]
Converted to discussion as this is unlikely a bug in `CefSharp`.

Did you test with the `MinimalExample` and confirm the issue reproduces there?

https://github.com/cefsharp/CefSharp.MinimalExample

> However, the problem does not occur in CefSharp.Wpf.NETCore version 124.3.80, so what is the difference?

https://github.com/cefsharp/CefSharp/issues/4835

Or it's simply just a `upstream` change in how `Chromium` behaves.

> Unhandled exception in FINALForensics.exe at 0x00007FFCB8B230C4 (rpcrt4.dll). 0xC0020043: rpc encountered an internal error (parameters: 0x0000000000000001, 0x00000000000006A6).

`rpcrt4.dll` is used by `Chromium`, not used directly by `CefSharp`, so we have no direct control over the behaviour.

You'll need to load the `libcef.dll.pdb` and get a meaningful stack trace.

https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#unmanaged-crashes-when-the-process-dies
